### PR TITLE
Conditionally include preload plugin code. NFC

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

**Review by dschuff — APPROVED** [review]
Code change looks ok, although I don't really know anything about what this code actually does. Is this a commonly-used feature? 
Could you add a commit message to say something about why this change is being made?

File: src/library_fs.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ Object.defineProperties(FSNode.prototype, {
 });
 FS.FSNode = FSNode;
 FS.staticInit();` +
-#if USE_CLOSURE_COMPILER
+#if USE_CLOSURE_COMPILER || USE_PRELOAD_PLUGINS
```
Comment (kripken) [inline src/library_fs.js:89]: If we don't use closure, why do we need this even with preload-plugins?

File: src/library_wasmfs.js [hunk 2]
```diff
@@ -7,7 +7,7 @@
 mergeInto(LibraryManager.library, {
   $wasmFSPreloadedFiles: [],
   $wasmFSPreloadedDirs: [],
-#if USE_CLOSURE_COMPILER
+#if USE_CLOSURE_COMPILER && USE_PRELOAD_PLUGINS
```
Comment (kripken) [inline src/library_wasmfs.js:10]: (same question)

**Comment by kripken** [comment]
I think this would be a very good simplication, but I think there might be more needed to get tests passing. I tried something related in #18408 and ran into some issues and stalled out there. If you want I can refresh my memory about what those issues were.

**Comment by sbc100** [comment]
Abandoning after I discovered that the preload system can be used even without the file packaged and without the `--use-preload-plugin` flag.

**Pull request closed without merging**